p2p/antiflood/p2pAntiflood.go

```diff
@@ -50,6 +54,24 @@ func (af *p2pAntiflood) CanProcessMessage(message p2p.MessageP2P, fromConnectedP
 	return nil
 }
 
+// CanProcessMessageOnTopic signals if a p2p message can be processed or not for a given topic
+func (af *p2pAntiflood) CanProcessMessageOnTopic(message p2p.MessageP2P, fromConnectedPeer p2p.PeerID, topic string) error {
```

Comment:
I would just leave the function signature as 
```GO
func (af *p2pAntiflood) CanProcessMessageOnTopic(peer p2p.PeerID, topic string) error {
```
as the message payload size is computed on the other flood preventer. It would be clearer how to use it

Reply:
right now I use the message to get the peer ID. Should I use the fromConnected to ge the peer and send it to this function ?

Reply:
Depends, if used inside worker and heartbeat, you should take the peer ID from the message. Inside the header interceptor, you should use the fromConnected peer.

Reply:
done